CONTRIBUTING.md

```diff
@@ -74,23 +74,45 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
 
 1. **Set Remote** upstream to be `https://github.com/dragonflyoss/Dragonfly.git` using the following two commands:
 
-```
-git remote add upstream https://github.com/dragonflyoss/Dragonfly.git
-git remote set-url --push upstream no-pushing
-```
+	```
+	git remote add upstream https://github.com/dragonflyoss/Dragonfly.git
+	git remote set-url --push upstream no-pushing
+	```
 
-With this remote setting, you can check your git remote configuration like this:
+	With this remote setting, you can check your git remote configuration like this:
 
-```
-$ git remote -v
-origin     https://github.com/<your-username>/Dragonfly.git (fetch)
-origin     https://github.com/<your-username>/Dragonfly.git (push)
-upstream   https://github.com/dragonflyoss/Dragonfly.git (fetch)
-upstream   no-pushing (push)
-```
+	```
+	$ git remote -v
+	origin     https://github.com/<your-username>/Dragonfly.git (fetch)
+	origin     https://github.com/<your-username>/Dragonfly.git (push)
+	upstream   https://github.com/dragonflyoss/Dragonfly.git (fetch)
+	upstream   no-pushing (push)
+	```
 
 Adding this, we can easily synchronize local branches with upstream branches.
```

Comment:
This line should be indented 4 spaces to the right. You can view the formatted doc in your branch [here](https://github.com/truongnh1992/Dragonfly/blob/contribution/CONTRIBUTING.md#workspace-preparation).

Reply:
Thank @lowzj. I've just updated.